### PR TITLE
Prepare project for updated AirConsole API 1.9.0 usage

### PR DESCRIPTION
**Pull request by marc-n-dream**
I think the biggest value in the code review is 'sanity checking' the CHANGES.md and upgrade migration instruction in README.md

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<!-- markdownlint-disable MD024 -->
```
Comment (daniel-n-dream) [inline CHANGES.md:1]: CHANGELOG.md seems like a standard, why change it?

File: CHANGES.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+- Support for custom local IPs: AirConsole now provides a field to define the IP provided to the backend, enabling the AirConsole Controller to connect to running Unity Editor instances directly.
+
+## [2.13] - 2022-10-23
+
+### Fixed
+
+- Update Unity Webview to v1.0.1 to address WebGL builds
+
+## [2.12] - 2023-10-10
```
Comment (daniel-n-dream) [inline CHANGES.md:45]: Date seems wrong? What are they corresponding to exactly anyway?

File: Assets/AirConsole/examples/basic/controller.html [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <html>
 <head>
   <meta name="viewport" content="user-scalable=no, width=device-width, initial-scale=1.0, maximum-scale=1.0"/>
-  <script type="text/javascript" src="https://www.airconsole.com/api/airconsole-1.8.0.js"></script>
+  <script type="text/javascript" src="https://www.airconsole.com/api/airconsole-1.9.0.js"></script>
```
Comment (daniel-n-dream) [inline Assets/AirConsole/examples/basic/controller.html:5]: I wonder why we don't use api/airconsole-latest?

File: README.md [hunk 4]
```diff
@@ -15,9 +15,12 @@ You don't need to install any other webserver or services.
 You need to copy the correct version from `WebGLTemplates/AirConsole` for Unity 2019 and below or `WebGLTemplates/AirConsole-2020` for Unity 2020+ to the WebGLTemplate folder you use.
 
 2. The content of the index.html / screen.html has changed.
-Search for `<script src="translation.js"></script>` and replace it with `<script src="airconsole-settings.js"></script>`, otherwise neither Translations nor Player Silencing will work.
+Open your controller's html file as well as the index.html in your WebGL template and search for `<script src="translation.js"></script>` and replace it with `<script src="airconsole-settings.js"></script>`, otherwise neither Translations nor Player Silencing will work.
 
-3. Ensure that obsolete API devices, device_id and server_time_offset are updated:
+3. The version of the airconsole api has changed.
+Search for a script usage like `<script src="https://www.airconsole.com/api/airconsole-1.8.0.js"></script>` and replace it with `<script src="https://www.airconsole.com/api/airconsole-1.9.0.js"></script>`
```
Comment (daniel-n-dream) [inline README.md:21]: Same here, wouldn't it be better to tell them to always use the latest api version?

File: Assets/WebGLTemplates/AirConsole-2020/airconsole-settings.js [hunk 5]
```diff
@@ -1,2 +1,2 @@
-var AIRCONSOLE_TRANSLATION = false;
-var AIRCONSOLE_INACTIVE_PLAYERS_SILENCED = false;
\ No newline at end of file
+var AIRCONSOLE_TRANSLATION = true;
```
Comment (daniel-n-dream) [inline Assets/WebGLTemplates/AirConsole-2020/airconsole-settings.js:1]: Are those the default values?

File: Assets/WebGLTemplates/AirConsole-2020/airconsole-unity-plugin.js [hunk 6]
```diff
@@ -355,8 +361,6 @@ App.prototype.processUnityData = function (data) {
         this.airconsole.setCustomDeviceState(data.data);
     } else if (data.action == "setCustomDeviceStateProperty") {
         this.airconsole.setCustomDeviceStateProperty(data.key, data.value);
-    } else if (data.action === "enablePlayerSilencing") {
```
Comment (daniel-n-dream) [inline Assets/WebGLTemplates/AirConsole-2020/airconsole-unity-plugin.js:358]: Is it normal that it is removed?

File: CHANGES.md (hunk 2 above)
Comment (marc-n-dream) [reply CHANGES.md:45]: That should be the date when the respective version tag was created.

The goal: Automate creation of Github release using the changelog file through a github action. I've such systems with this changelog format already from past efforts, so it should be relatively straight forward.

File: Assets/WebGLTemplates/AirConsole-2020/airconsole-unity-plugin.js (hunk 6 above)
Comment (marc-n-dream) [reply Assets/WebGLTemplates/AirConsole-2020/airconsole-unity-plugin.js:358]: Yes. It never was in use - it was originally added during the initial phase of the 2.5.0 efforts but we ultimately decided on going a different path (AirConsole init time where the entry in the airconsole-settings.js stems from) removing any runtime API.

**Pull request merged**